08-read-write-plot.qmd

```diff
@@ -291,6 +292,44 @@ The next two sections will demonstrate how to do this.
 
 ### Vector data
 
+The counterpart of `gpd.read_file` is the `.to_file` method that a `GeoDataFrame` has. It allows you to write `GeoDataFrame` objects to a wide range of geographic vector file formats, including the most common, such as `.geojson`, `.shp` and `.gpkg`. Based on the file name, `.to_file` decides automatically which driver to use. The speed of the writing process depends also on the driver.
```

Comment:
:+1: to this clear description.

Reply:
Thanks, it's adapted from the R book, so credit goes to you @Robinlovelace and @Nowosad :-)